compiler/cpp/src/thrift/generate/t_swift_generator.cc

```diff
@@ -245,22 +297,30 @@ class t_swift_generator : public t_oop_generator {
  */
 void t_swift_generator::init_generator() {
   // Make output directory
-  MKDIR(get_out_dir().c_str());
+  string module = get_real_swift_module(program_);
+  string out_dir = get_out_dir();
+  string module_path = out_dir;
+  string name = capitalize(program_name_);
+  if (namespaced_ and !module.empty()) {
```

Comment:
I just noticed that this broke Windows build.
It seems that VC++ does not support alternative keywords in our settings.
 Can you replace`and` with `&&` ? We'll see if it fixes the build.